kyo-core/shared/src/main/scala/kyo/aborts.scala

```diff
@@ -46,6 +47,15 @@ object Aborts:
                     fail(ex.asInstanceOf[V])
             }
 
+        def trying[T, S](v: T < (Aborts[V] & S))(
```

Comment:
This function is useful, but perhaps unnecessary. We can also just `.map(_.toTry)` at the call site of `Aborts[Throwable].run`.

Reply:
I'd prefer not to add it. It doesn't seem it'd be a common operation and there's `IOs.attempt` that is similar.